@Old-Ding Please fix

```
a06291a2d9 tools: escape Python string backslashes
../nuttx/tools/checkpatch.sh -c -u -m -g  8b49e46b9ee7306a463107aad643c93d5b604309..HEAD
All done! ✨ 🍰 ✨
1 file would be left unchanged.
Used config files:
    1: .codespellrc
All done! ✨ 🍰 ✨
1 file would be left unchanged.
would reformat /home/runner/work/nuttx/nuttx/nuttx/tools/licensing/check.py

Oh no! 💥 💔 💥
1 file would be reformatted.

black check failed, run following command to update the style:
  $ black <src>

Some checks failed. For contributing guidelines, see:
  https://github.com/apache/nuttx/blob/master/CONTRIBUTING.md
Error: Process completed with exit code 1.
```

--- a/tools/ide_exporter.py
+++ b/tools/ide_exporter.py
@@ -474,7 +474,7 @@ class IARProject(IdeProject):
         IAR sample including nodes
             <option>
               <name>CCIncludePath2</name>
-              <state>$PROJ_DIR$\nuttx\include\</state>
+              <state>$PROJ_DIR$\\nuttx\\include\\</state>
             </option>
         Args:
             sources: list of SourceInfo

--- a/tools/licensing/apachize.py
+++ b/tools/licensing/apachize.py
@@ -34,7 +34,7 @@ def apachize(path, header):
 
     with open(path) as f:
         s = f.read()
-        s = re.sub("(?i)/\*\*\*.+?(?:Copyright).+?\*\*\*+/", header, s, 1, re.DOTALL)
+        s = re.sub(r"(?i)/\*\*\*.+?(?:Copyright).+?\*\*\*+/", header, s, 1, re.DOTALL)
         print(s)
 
 

--- a/tools/licensing/check.py
+++ b/tools/licensing/check.py
@@ -51,7 +51,7 @@ def commit_attributions(c):
 
 
 def get_headers(s):
-    return re.findall("(?i)/\*\*\*.+?(?:Copyright).+?\*\*\*+/", s, re.DOTALL)
+    return re.findall(r"(?i)/\*\*\*.+?(?:Copyright).+?\*\*\*+/", s, re.DOTALL)
 
 
 def get_file(blob):
@@ -131,7 +131,7 @@ def author_has_cla(author):
 
 def header_copyrights(header):
     results = re.findall(
-        " \* *[Cc]opyright:?(?: ?.[Cc].)? *(?:[12][0-9]{3}[,-]? ?)* *(.+)", header
+        r" \* *[Cc]opyright:?(?: ?.[Cc].)? *(?:[12][0-9]{3}[,-]? ?)* *(.+)", header
     )
     return [re.sub("(. )?[Aa]ll rights reserved.?", "", result) for result in results]
 
@@ -139,7 +139,7 @@ def header_copyrights(header):
 def report_cla(author):
     cla = author_has_cla(author)
     if cla:
-        (apacheid, name) = cla
+        apacheid, name = cla
         print(colored("✓", "green"), end=" ")
     else:
         apacheid = None

